canonical-web-and-design/vanilla-framework
# Move subnav styles to main nav styles

## Done

- Renamed `.p-subnav` pattern to `.p-navigation__item--dropdown-toggle` and made it a child of the main `.p-navigation` pattern
- Deprecated the existing example

Fixes #2857 

## QA

- Open the following:
  - [Dropdown](https://vanilla-framework-3765.demos.haus/docs/examples/patterns/navigation/dropdown)
  - [Dark dropdown](https://vanilla-framework-3765.demos.haus/docs/examples/patterns/navigation/dropdown-dark)
  - [Deprecated subnav](https://vanilla-framework-3765.demos.haus/docs/examples/patterns/navigation/deprecated-subnav)
  - [Deprecated dark subnav](https://vanilla-framework-3765.demos.haus/docs/examples/patterns/navigation/deprecated-subnav-dark)
- Check that they all work, and look indentical
- Review updated documentation:
  - [Component status](https://vanilla-framework-3765.demos.haus/docs/component-status)
  - [Upgrade docs](https://vanilla-framework-3765.demos.haus/docs/upgrade-guide-v3)
  - [Navigation docs](https://vanilla-framework-3765.demos.haus/docs/patterns/navigation#dropdown)

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
